jsonschema-validation.xml

```diff
@@ -172,15 +172,26 @@
 
             <section title="Validation of primitive types and child values">
                 <t>
-                    Two of the primitive types, array and object, allow for child values.  The validation of
-                    the primitive type is considered separately from the validation of child instances.
+                    Two of the primitive types, array and object, allow for child values.
+                    The validation of the primitive type is considered separately from
+                    the validation of child instances.
                 </t>
                 <t>
-                    For arrays, primitive type validation consists of validating restrictions on length.
+                    For arrays, primitive type validation consists of validating
+                    restrictions on length with "minItems" and "maxItems", while
+                    "items" and "additionalItems" control which subschemas apply
+                    to which elements of the array.  In addition, "uniqueItems"
```

Comment:
It's extremely important because implementing these is different from implementing other keywords.  There are basically four types of validation keywords:

* Immediate instance keywords (most of them)
* Apply-subschema-to-child-instance keywords (these)
* Logical combinations of schema results ("allOf"/"anyOf"/"oneOf"/"not"/"dependencies"/"if"-"then"-"else" assuming it eventually goes in)
* Array content keywords ("uniqueItems" and "contains") that can only be evaluated by looking at the entire array and all of its contents